llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -805,6 +805,13 @@ def FeatureStdExtZvksc
 def FeatureStdExtZvksg
     : RISCVExtension<1, 0, "shorthand for 'Zvks' and 'Zvkg'",
                      [FeatureStdExtZvks, FeatureStdExtZvkg]>;
+
+def FeatureStdExtZvzip
+   :  RISCVExperimentalExtension<0, 1, "zip/unzip/zipeven/zipodd">;
```

Comment:
zip->vzip or add to match the instruction names and indicate vectorness. There's a scalar zip/unzip instruction in Zvkb.